src/protocols.json

```diff
@@ -119,6 +127,14 @@
       "imageLight": "yeet.svg",
       "name": "YEET",
       "url": "https://yeetit.xyz"
+    },
+    {
+      "description": "ZeroLend is a lending protocol on Berachain. Powered by Euler.",
+      "id": "zerolend",
+      "imageDark": "zerolend.svg",
+      "imageLight": "zerolend.svg",
+      "name": "Zerolend",
+      "url": "https://zerolend.xyz"
```

Comment:
Is this for future tokens? It's currently unused.

If so, please remove it for now and add it on future PRs.